modules/bridBidAdapter.js

```diff
@@ -0,0 +1,224 @@
+import {createTrackPixelHtml, _each, deepAccess, getDefinedParams, parseGPTSingleSizeArrayToRtbSize} from '../src/utils.js';
+import {VIDEO} from '../src/mediaTypes.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {getRefererInfo} from '../src/refererDetection.js';
+
+const SOURCE = 'pbjs';
+const BIDDER_CODE = 'brid';
+const ENDPOINT_URL = 'https://pbs.prebrid.tv/openrtb2/auction';
+const GVLID = 934;
+const TIME_TO_LIVE = 300;
+const VIDEO_PARAMS = [
+  'api', 'linearity', 'maxduration', 'mimes', 'minduration', 'placement',
+  'playbackmethod', 'protocols', 'startdelay'
+];
+
+export const spec = {
+
+  code: BIDDER_CODE,
+  gvlid: GVLID,
+  supportedMediaTypes: [VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid && bid.params && bid.params.placementId);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} bidRequests A non-empty list of bid requests which should be sent to the Server.
+   * @param {BidderRequest} bidderRequest bidder request object.
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(bidRequests, bidderRequest) {
+    const requests = [];
+
+    _each(bidRequests, function(bid) {
+      const placementId = bid.params.placementId;
+      const auctionId = bid.auctionId;
+      const bidId = bid.bidId;
+      const params = bid.params;
+      let sizes = bid.sizes;
+      if (sizes && !Array.isArray(sizes[0])) sizes = [sizes];
+
+      const site = getSiteObj();
+
+      const postBody = {
+        sdk: {
+          source: SOURCE,
+          version: '$prebid.version$'
+        },
+        id: auctionId,
```

Comment:
So this looks like openrtb, and in openrtb $.id should be unique to a request, not shared with other requests, so you cannot use auctionid here. You should instead use bidderrequestid. Auctionid is most fitting for source.tid, however, it might be null in the bid request object as of prebid 8, so that publishers can control downstream information leaks. This is why my previous comments said to make sure your adapter can take null auctionid.

Reply:
Hi @patmmccann , thank you for the explanation! Removed auctionId from requests and changed id to use bidderRequest.bidderRequestId instead.